web/app/js/index.js

```diff
@@ -40,6 +41,7 @@ let applicationHtml = (
             <Switch>
               <Redirect exact from={`${pathPrefix}/`} to={`${pathPrefix}/servicemesh`} />
               <Route path={`${pathPrefix}/servicemesh`} render={() => <ServiceMesh api={api} releaseVersion={appData.releaseVersion} controllerNamespace={controllerNs} />} />
+              <Route path={`${pathPrefix}/graph`} render={() => <NetworkGraph api={api} controllerNamespace={controllerNs} />} />
```

Comment:
you'll need to also add the `/graph` path in `web/server.go`, otherwise navigating directly to `/graph` will 404